### PR TITLE
Update to sabre 2.1.3

### PR DESCRIPTION
**Pull request by PVince81**
@DeepDiver1975 @LukasReschke

### CODE REVIEW

**Comment by PVince81** [comment]
- pyocclient test suite passes
- smashbox test_basicsync has errors on master already, so cannot use it to confirm this PR, will investigate.

**Comment by PVince81** [comment]
I was running it wrong. After clearing the smashdir, the smashbox test "test_basicsync" also works on this branch.

File: sabre/dav/.travis.yml [hunk 1]
```diff
@@ -8,17 +8,22 @@ php:
 matrix:
   fast_finish: true
 
+env:
```
Comment (nickvergessen) [inline sabre/dav/.travis.yml]: This file should not be in our repo?

Comment (PVince81) [inline sabre/dav/.travis.yml]: True, will remove. Also I see that the bin folder has sneaked in back, will remove it too.

**Comment by PVince81** [comment]
Added travis files to gitignore: https://github.com/owncloud/3rdparty/commit/7b3940dd4ce3b5fd16bd35418737de285adb69b6

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**